pkg/e2e/fixtures/build-test/multi-args/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG IMAGE=666
+ARG TAG=666
+
+FROM ${IMAGE}:${TAG}
```

Comment:
We need a way to input two different values for the args, and afterwards verify that they’re still two different values.